docs/Rimpl.md

```diff
@@ -1,17 +1,18 @@
 # Rimpl
 
-# Introduction
+## Introduction
 
 Rimpl is a command line tool that computes [Impl (Impact YAML)](Impl%20(Impact%20YAML).md) files. 
-# Help
+
+## Quickstart
 
 ```
 rimpl 
 -impl [path to the input impl file]
 -ompl [path to the output impl file]
--format [yaml|csv] 
--verbose
--help 
+-format [yaml|csv] (not yet implemented)
```

Comment:
@jawache do we need `format` flag?

Reply:
let's remove for now as it is not implemented - we can revisit if this feature is added later.
```suggestion
```